presidio-analyzer/presidio_analyzer/batch_analyzer_engine.py

```diff
@@ -28,13 +28,15 @@ def analyze_iterator(
         self,
         texts: Iterable[Union[str, bool, float, int]],
         language: str,
+        batch_size: Optional[int] = None,
         **kwargs,
     ) -> List[List[RecognizerResult]]:
         """
         Analyze an iterable of strings.
 
         :param texts: An list containing strings to be analyzed.
         :param language: Input language
+        :param batch_size: Batch size to process in a single iteration
```

Comment:
What happens if the batch size is 0?

Reply:
Or how do I keep previous behavior if I send kwargs? 
batch size should be -1?

Reply:
I think it depends on the implementation of the nlp engine, which is injected by the user.

Reply:
Maybe add that to the documentation? 
batch_size - depends on nlp engine implementation?
@omri374 WDYT?